cmd/gsw_service.go

```diff
@@ -58,6 +61,23 @@ func decomInitialize(ctx context.Context) map[int]chan []byte {
 	return channelMap
 }
 
+func dbInitialize(ctx context.Context, channelMap map[int]chan []byte) {
```

Comment:
might want to return this error so you can respond accordingly. The database not initializing seems like something you need to care about rather than just logging it and continuing life.

Reply:
We can return the error here, but won't actually fatal from finishing boot. I consider logging to DB to not be a hard requirement for running GSW. In V1, this was originally an app you ran instead.

Reply:
Also funny enough, if your influx service isn't running, dbInitialize still passes lol. The Influx V1 implementation uses UDP so you can still dial into port 8089, but when you send itll fail until the Influx service goes up